devicetypes/smartthings/orvibo-motion-sensor.src/Orvibo-Motion-Sensor.groovy

```diff
@@ -0,0 +1,167 @@
+			tileAttribute("device.motion", key: "PRIMARY_CONTROL") {
+				attributeState "active", label: 'motion', icon: "st.motion.motion.active", backgroundColor: "#00A0DC"
+				attributeState "inactive", label: 'no motion', icon: "st.motion.motion.inactive", backgroundColor: "#cccccc"
+			}
+		}
+		valueTile("battery", "device.battery", decoration: "flat", inactiveLabel: false, width: 2, height: 2) {
+			state "battery", label: '${currentValue}% battery', unit: ""
+		}
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+		main(["motion"])
+		details(["motion","battery", "refresh"])
+	}
+}
+def collectAttributes(Map descMap) {
+	def descMaps = new ArrayList<Map>()
+	descMaps.add(descMap)
+	if (descMap.additionalAttrs) {
+		descMaps.addAll(descMap.additionalAttrs)
+	}
+	return  descMaps
+}
+def stopMotion() {
+	log.debug "motion inactive"
+	sendEvent(getMotionResult('inactive'))
+}
+def installed(){
+	log.debug "installed"
+	return zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021) +
+					zigbee.readAttribute(zigbee.IAS_ZONE_CLUSTER,zigbee.ATTRIBUTE_IAS_ZONE_STATUS)
+
+}
+def parse(String description) {
+	log.debug "description(): $description"
+	def map = zigbee.getEvent(description)
+	if(!map){
+		if (description?.startsWith('zone status')) {
+			map = parseIasMessage(description)
+			motionHandler(description);
+		} else {
+			map = batteyHandler(description);
+		}
+	}
+	log.debug "Parse returned $map"
+	def result = map ? createEvent(map) : [:]
+	if (description?.startsWith('enroll request')) {
+		List cmds = zigbee.enrollResponse()
+		log.debug "enroll response: ${cmds}"
+		result = cmds?.collect { new physicalgraph.device.HubAction(it) }
+	}
+	return result
+}
+def batteyHandler(String description){
+	def descMap = zigbee.parseDescriptionAsMap(description)
+    def map
+	if (descMap?.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER && descMap.commandInt != 0x07 && descMap.value) {
+		log.info "BATT METRICS - attr: ${descMap?.attrInt}, value: ${descMap?.value}, decValue: ${Integer.parseInt(descMap.value, 16)}, currPercent: ${device.currentState("battery")?.value}, device: ${device.getDataValue("manufacturer")} ${device.getDataValue("model")}"
+		List<Map> descMaps = collectAttributes(descMap)
+		def battMap = descMaps.find { it.attrInt == 0x0021 }
+		if (battMap) {
+			map = getBatteryPercentageResult(Integer.parseInt(battMap.value, 16))
+		}
+	}
+	return map;
+}
+def motionHandler(String description){
+	//inactive
+	def isActive = zigbee.translateStatusZoneType19(description)
+	def value = isActive ? "active" : "inactive"
+	if (value == "active") {
+		def timeout = 3
+		log.debug "Stopping motion in ${timeout} seconds"
+		runIn(timeout, stopMotion)
+	}
+}
+def parseIasMessage(String description) {
+	ZoneStatus zs = zigbee.parseZoneStatus(description)
+	translateZoneStatus(zs)
```

Comment:
You are creating new method (translateZoneStatus) just to call return without any additional logic. Just call return here.
Or even better. Change getMotionResult(value) method. Check if value is true or false and create events based on that. Remeber to change stopMotion().